superflore/generators/bitbake/run.py

```diff
@@ -93,7 +93,8 @@ def main():
         overlay = RosMeta(
             _repo,
             not args.output_repository_path,
-            branch='superflore/{}'.format(now),
+            branch=(('superflore/{}'.format(now)) if not args.no_branch
+                                                  else None),
```

Comment:
@shr-project the linter is unhappy with this line

Reply:
Looking into it now (before the line was too long now this.. :))

Reply:
> before the line was too long now this.. :)

Haha, yeah. Welcome to superflore changes ;). Linter is pretty picky.